cli-plugins/accelerator/command-reference/tanzu_accelerator_generate_from_local.hbs.md

```diff
@@ -0,0 +1,50 @@
+## `tanzu accelerator generate-from-local`
```

Comment:
@eleftherias - please review and let me know if anything should be revised.

Reply:
I believe we should also update tanzu_accelerator.hbs.md with a reference to the generate-from-local docs https://github.com/pivotal/acc-tanzu-cli/blob/main/docs/tanzu_accelerator.md

Reply:
ahhh good point, I'll update that. Thanks!